requirements/dev.txt

```diff
@@ -2,3 +2,5 @@
 -r base.txt
 
 money-to-prisoners-common[testing]>=9.16,<9.17
+boto3>=1.11
```

Comment:
since the command that uses boto could be useful to run both locally and remotely, this line ought to go into base.txt: that way it's available to everyone